protocol_tests/__init__.py

```diff
@@ -222,3 +225,26 @@ def temporary_channel(
     """
     async for _event, *_data in generator:
         pass
+
+
+class BackChannel:
```

Comment:
The topic of backchannels was discussed at length at the Connectathon and have not yet reached a conclusion. In light of discussions pending, loading in backchannels dynamically has been adopted as a temporary solution, allowing test development to continue without the backchannel being completely defined yet. I'd like to see these changes reconciled with the pluaggable backchannel mechanism.